ERCS/erc-6932.md

```diff
@@ -0,0 +1,122 @@
+
+This EIP proposes the standardization of a subscription token on the Ethereum blockchain, enabling automatic periodic deductions from user balances as determined by the merchant subscriber. The Subscription Token aims to simplify and streamline subscription-based services on the Ethereum network, offering enhanced convenience and efficiency for users and merchants alike.
+
+We will be also adding `renewSubscription` functions to the token contract, that can be used by token holders to renew their subscription to a service or product that requires recurring payments in the form of the token.
+
+## Motivation
+
+The rise of subscription-based business models necessitates a standardized approach to handle recurring payments on the Ethereum blockchain. Currently, users often manually initiate subscription payments, resulting in inconvenience and potential disruptions in service delivery. By introducing a Subscription Token, users can seamlessly authorize periodic deductions, enabling uninterrupted access to subscribed services.
+
+The subscription-based [ERC-20](./eip-20.md) token provides a more flexible and convenient way to manage recurring payments. It can be used for a wide range of services and products that require regular payments, such as subscription-based content platforms, gaming services, and more.
+
+## Specification
+
+The Subscription Token ensures consistency and interoperability across different implementations. Key features include:
+
+- Auto Deduction: Merchants, acting as subscribers, can set the subscription interval and associated payment amount for their services. This information is encoded within the Subscription Token contract, enabling automatic deductions from user balances at regular intervals without requiring manual intervention.
+
+- Balance Check: Users can verify the remaining balance of their subscription tokens at any given time. This transparency empowers users to monitor their subscriptions and make informed decisions regarding their ongoing commitment to the service.
+
+- Flexibility: The Subscription Token framework accommodates various subscription models, such as monthly, quarterly, or annual billing cycles. Additionally, merchants have the option to define trial periods, upgrade/downgrade plans, and cancellation policies, providing a versatile foundation for a wide range of subscription-based businesses.
+
+- Security: The Subscription Token employs established security measures, including the use of cryptographic signatures, to ensure the integrity and authenticity of subscription-related transactions. This protects both users and merchants from unauthorized access and potential malicious activities.
+
+### SubscriptionToken
```

Comment:
Please put code snippets (like contract/function identifiers) in backticks (\`). This goes for all instances, not just this line.

```suggestion
### `SubscriptionToken`
```